src/server-schema.graphql

```diff
@@ -1089,6 +1089,8 @@ input QueryAllBusinessesParameters {
   where: QueryAllBusinessesConditions
   order: QueryAllBusinessesOrder
   ownCoordinates: Geolocation
+  limit: Int
```

Comment:
i wonder if it may also make sense to implement the 'backend' side like for pagination

      page: pageNumber,
      limit: pageSize,
      startIndex,
      endIndex

const startIndex = (pageNumber - 1) * pageSize;
 const endIndex = startIndex + pageSize;

This way, if we want at some point use pagination instead, we can (for example if we want to be able to send a url that specifically points to a certain area within the search results).

And we can use it this way also for lazy loading.